### DISCUSSION TITLE
Defaults command not found.

### DISCUSSION THREAD

**Discussion by Treach96** [holzschu/a-shell#772] [Q&A] [answered]
Hello together, 
I’m total new to this kind of topic so I need your help:D I want to add some css for obsidian on my iPad but I have no access to .obsidian because it’s hidden.
Now I found a tutorial which said if I use the a shell as terminal I can access the file. I managed to find it but I can’t make it visible in the file system. Further it said if I use ´defaults com.apple.files ShowAllFiles True ´ I can change the setting. But i only get the response „defaults: command not found“. Do I need to install it manually in the terminal? In which package can I find it?
Also i found a solution which said to use /usr/bin defaults in case the path is wrong but the access get denied.

TLDR: command defaults can’t be found for „defaults com.apple.files ShowAllFiles“ what should I do?
 Edit 1: typo

**Comment by holzschu** [comment, marked as answer]
The command "defaults" only exists on Macs, not on iPads or iPhones (and the mechanism it uses is not available either).

If the .obsidian folder is inside a folder that you can access with pickFolder, then you can use shell commands like `ls -a`, `cp`... to move files inside it.

**Reply by Treach96** [reply]
Thanks a lot! Then I'm trying it this way.
